routes/projects/files/get-one.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * GET /projects/:username/:projectName/files/:pathToFile
+ * The :pathToFile parameter must be URI encoded
+ * Returns the contents of the given file in the form:
+ * {
+ *   type: <string> content-type,
+ *   path: <string>,
+ *   link: <string>
+ * }
```

Comment:
I wonder if it's useful to also send `size` so that clients can judge the amount of work to download various resources (e.g., progress updates for opening a project).